clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -1336,13 +1335,13 @@ public ClientResponse mockOffsetCommitResponse(String topic,
                                                    long receivedTimeMs,
                                                    Errors error) {
         OffsetCommitResponseData responseData = new OffsetCommitResponseData()
-            .setTopics(Arrays.asList(
-                new OffsetCommitResponseData.OffsetCommitResponseTopic()
-                    .setName(topic)
-                    .setPartitions(Collections.singletonList(
-                        new OffsetCommitResponseData.OffsetCommitResponsePartition()
-                            .setErrorCode(error.code())
-                            .setPartitionIndex(partition)))));
+            .setTopics(Collections.singletonList(
+                    new OffsetCommitResponseData.OffsetCommitResponseTopic()
```

Comment:
Could you please remove those unrelated changes? That can reduce the size of PR and avoid unnecessary conflicts to other PRs

Reply:
Done! Thanks for pointing out.